src/Typechecker/Monad.hs

```diff
@@ -188,3 +191,4 @@ instance Show TypeError where
   show (Unknown s p)           = errString p ++ s
   show (BadOp o t1 t2 e p)     = errString p ++ "Cannot '" ++ show o ++ "' types " ++ show t1 ++ " and " ++ show t2 ++ " in expression:\n\t" ++ show e
   show (OutOfBounds x y p)     = errString p ++ "Could not access (" ++ show x ++ "," ++ show y ++ ") on the board, this is not a valid space. "
+  show (Uninitialized n p)     = errString p ++ "Incomplete initialization of Board '" ++ n ++ "'"
```

Comment:
You should use the [function for quoting strings](https://github.com/The-Code-In-Sheep-s-Clothing/Spiel-Lang/blob/c6ce34d5bd7b0515dcc96c8dfb280532b7b677b6/src/Parser/Error.hs#L5). Admittedly, it is tucked away in a bad spot. Perhaps we should have a Utilities module?

Reply:
This might go well in a Utils.String module actually.